src/style.css

```diff
@@ -0,0 +1,33 @@
+.body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row-reverse;
```

Comment:
why reverse?

Reply:
I used it for mor comfottable work with `nth-child`. But I have corected this for more intuitive understanding of my code.

Reply:
In next commit*